docs/parameters_desc.md

```diff
@@ -0,0 +1,189 @@
+This document attempts to describe the important parameters in Macrobase that
```

Comment:
Attempts? ;) 

"This document describes MacroBase configuration and query parameters."